crud/select/merger.lua

```diff
@@ -249,7 +256,8 @@ local function new_readview(vshard_router, replicasets, readview_uuid, space, in
                 if replica_uuid == value.uuid then
                     -- Perform a request.
                     local buf = buffer.ibuf()
-                    local net_box_opts = {is_async = true, buffer = buf, skip_header = false}
+                    local net_box_opts = {is_async = true, buffer = buf,
+                                          skip_header = utils.tarantool_supports_netbox_skip_header_option()}
```

Comment:
In case we didn't support it we still add unknown option `{skip_header = false}`. Let's set this option to non-nil only if supported.

Reply:
Fixed.